graph.go

```diff
@@ -12,6 +12,7 @@ import (
 
 const (
 	// weightNormal is the typcal edge weight.
+	//nolint:unused
 	weightNormal = 1
```

Comment:
Ignoring, since it might not be used but is documentational "enum" value at the very least.
```
graph.go:15:2: const weightNormal is unused (unused)
        weightNormal = 1
```